3-enrich/stream-enrich/examples/config.hocon.sample

```diff
@@ -69,6 +69,13 @@ enrich {
       # customEndpoint = {{kinesisEndpoint}}
       # customEndpoint = ${?ENRICH_STREAMS_SOURCE_SINK_CUSTOM_ENDPOINT}
 
+      ## Optional endpoint url configuration to override aws dyanomdb endpoints for Kinesis checkpoints lease table,
+      ## this can be used to specify local endpoints when using Localstack
+      # dynamodbCustomEndpoint = "http://localhost:4569"
```

Comment:
Shouldn't we offer the ability to provide this field through an environment variable? Like any other field in this config template.

Reply:
It's already the case. Once the key `dynamodbCustomEndpoint` is read in the code it can be specified either with direct value e.g.
```
dynamodbCustomEndpoint = "http://localhost:4569"
```
or with an environment variable e.g.
```
dynamodbCustomEndpoint = ${?WHATEVER_ENV_VAR}
```